Common/include/option_structure.hpp

```diff
@@ -2089,6 +2092,8 @@ enum PERIODIC_QUANTITIES {
   PERIODIC_LIM_PRIM_1 ,  /*!< \brief Primitive limiter communication phase 1 of 2 (periodic only). */
   PERIODIC_LIM_PRIM_2 ,  /*!< \brief Primitive limiter communication phase 2 of 2 (periodic only). */
   PERIODIC_IMPLICIT   ,  /*!< \brief Implicit update communication to ensure consistency across periodic boundaries. */
+  PERIODIC_AUX_GG,
+  PERIODIC_AUX_LS,
```

Comment:
add a brief, and list these close to other GG gradient things, if you are not using AUX_LS remove it.